src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConsoleSnippetProvider.cs

```diff
@@ -108,7 +102,46 @@ private async Task<TextChange> GenerateSnippetTextChangeAsync(Document document,
             return root.ReplaceNode(snippetExpressionNode, reformatSnippetNode);
         }
 
-        private static SyntaxNode? GetConsoleExpressionStatement(ISyntaxFactsService syntaxFacts, SyntaxNode root, int position)
+        protected override List<(string, List<TextSpan>)> GetRenameLocationsMap(SyntaxNode node, ISyntaxFacts syntaxFacts, CancellationToken cancellationToken)
```

Comment:
rename locations?

Reply:
poor name, it was before I changed the name of the property in the SnippetChange to Placeholders